docs/man/man1/kubectl-annotate.1

```diff
@@ -38,6 +38,10 @@ horizontalpodautoscalers (hpa), resourcequotas (quota) or secrets.
     Filename, directory, or URL to a file identifying the resource to update the annotation
 
 .PP
+\fB\-\-include\-extended\-apis\fP=true
```

Comment:
Reminder to @kubernetes/kubectl @kubernetes/rh-ux to review the flag name.  This is a global flag which controls whether or not the client will attempt to find third-party resource registrations on a given server.